packages/source-duckdb/src/index.ts

```
@@ -1,6 +1,11 @@
+export { flowbladeLogtapeDuckdbConfig } from './config/flowblade-logtape-duckdb.config';
+export type { DuckdbDatasourceParams } from './datasource/duckdb-datasource';
+export { DuckdbDatasource } from './datasource/duckdb-datasource';
+export { duckdbDefaultLogger } from './logger/duckdb-default-logger';
+
+// re-export from @flowblade/sql-tag
 export { sql, type SqlTag } from '@flowblade/sql-tag';
 
-// Export from @flowblade/core
+// re-export from @flowblade/code
 export type { QError } from '@flowblade/core';
 export { QResult } from '@flowblade/core';
```

ESLint

[error] Unable to resolve path to module '@flowblade/core'.

(import-x/no-unresolved)